### PR TITLE
Treetex proofs

### PR DESCRIPTION
**Pull request by AlCutter**
Make proofs in treetex output a bit clearer.

### Checklist

- [ ] I have updated the [CHANGELOG](CHANGELOG.md).
  - Adjust the draft version number according to [semantic versioning](https://semver.org/) rules.
- [X] I have updated [documentation](docs/) accordingly (including the [feature implementation matrix](docs/Feature_Implementation_Matrix.md)).

### CODE REVIEW

File: docs/merkletree/treetex/main.go [hunk 1]
```diff
@@ -186,9 +186,16 @@ func drawLeaf(prefix string, index uint64, leafText, dataText nodeTextFunc) {
 	a := nInfo[id]
 
 	// First render the leaf node of the Merkle tree
+	rangeIndices := a.rangeIndices
```
Comment (pavelkalinnikov) [inline docs/merkletree/treetex/main.go]: Could copy the whole `a` here, and use it for the second `Printf`.

Comment (AlCutter) [reply docs/merkletree/treetex/main.go]: good call, done.

File: docs/merkletree/treetex/main.go [hunk 2]
```diff
@@ -295,11 +300,18 @@ func modifyRangeNodeInfo() error {
 		// Set leaves:
 		for i := l; i < r; i++ {
 			id := compact.NewNodeID(0, i)
-			modifyNodeInfo(id, func(n *nodeInfo) { n.rangeIndices = append(n.rangeIndices, ri) })
+			modifyNodeInfo(id, func(n *nodeInfo) {
+				n.incPath = true
```
Comment (pavelkalinnikov) [inline docs/merkletree/treetex/main.go]: It's not an inclusion path, so maybe add a TODO to fix this?

Comment (AlCutter) [reply docs/merkletree/treetex/main.go]: I think this was actually unnecessary, dinged it.

**Pull request merged**